src/D2L.CodeStyle.Analyzers/Async/Generator/SyncGenerator.cs

```diff
@@ -20,7 +24,8 @@ internal sealed partial class SyncGenerator : IIncrementalGenerator {
 		// Do the generation per method
 		IncrementalValuesProvider<MethodGenerationResult> generatedMethods =
 			methodsToGenerate
-				.Select( GenerateSyncMethod )
+				.Combine( options )
+				.Select( (x, ct) => GenerateSyncMethod( x.Left.Item1, x.Left.Item2, x.Right, ct ) )

```

Comment:
`x.Left.Item1, x.Left.Item2, x.Right` pretty ugly lol

Reply:
Ah, I thought Combine magiced that. And you can't destructure into lambda variables yet so that's also sad